es_ES/layout.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: Tratando con las restricciones de cajas en Flutter
+
+permalink: /layout/
+---
+
+Usted puede ser redirigido a esta página si el maframework detecta un problema que involucre restricciones de cajas.
+
+
+* Placeholder for TOC
+{:toc}
+
+## Introducción
+
+En Flutter, los widgets son renderizados por sus objetos [`RenderBox`](https://docs.flutter.io/flutter/rendering/RenderBox-class.html) subyacentes. Los cuadros generadores obtienen restricciones por parte de sus padres y se clasifican dentro de esas restricciones. 
```

Comment:
Render boxes are given constraints by their parent, and size themselves within those constraints.
Render boxes  obtienen restricciones (_constraints_) de sus padres y ajustan su tamaño a estas restricciones.